controllers/useraccount/console_usersettings_test.go

```diff
@@ -0,0 +1,285 @@
+package useraccount
+
+import (
+	"context"
+	"fmt"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"testing"
+)
+
+func TestDeleteConsoleSettingObjects(t *testing.T) {
```

Comment:
minor - you could probably use _table driven tests_ for those , but feel free to ignore this or apply in follow up PR if you'd like to refactor the code